src/community/gsr/LICENSE.txt

```diff
@@ -0,0 +1,1416 @@
+GeoServer is distributed under the GNU General Public License Version 2.0 license:
```

Comment:
Please use the NOTICE.md and LICENSE.md as appropriate (avoiding use of the txt versions)

Reply:
fixed